cli/lib/wpt/result.mjs

```diff
@@ -132,9 +134,69 @@ function createGetMetricValue_( metric ) {
 			return ( run ) => run.firstView.bytesIn;
 	}
 
+	if ( metric.startsWith( 'Server-Timing:' ) ) {
+		const stMetric = metric.substring( 'Server-Timing:'.length );
+		if ( ! getServerTimingHeader ) {
+			getServerTimingHeader = createGetResponseHeader_( 'Server-Timing' );
+		}
+		return ( run ) => {
+			const stHeader = getServerTimingHeader( run );
+			const stIndex = stHeader.indexOf( `${ stMetric };dur=` );
+			if ( stIndex < 0 ) {
+				throw new Error( `Server-Timing metric ${ stMetric } not present in run` );
+			}
```

Comment:
If the command is run with the incorrect server-timing metric, it will not display this error, instead returning `0` in the median and `null` in runs (1-9).

**Command 1:**
```
npm run research -- wpt-metrics --test 221011_AiDcV7_GGM --metrics Server-Timing:wp-test
```

**Result:**

| Metric | Median |
| --- | --- |
| Server-Timing:wp-test | 0 |

**Command 2:**
```
npm run research -- wpt-metrics --test 221011_AiDcV7_GGM --metrics Server-Timing:wp-test --include-runs
```

**Result:**

| Metric | Median | Run 1 | Run 2 | Run 3 | Run 4 | Run 5 | Run 6 | Run 7 | Run 8 | Run 9 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Server-Timing:wp-test | 0 | null | null | null | null | null | null | null | null | null |

Reply:
Great catch! Fixed in https://github.com/GoogleChromeLabs/wpp-research/pull/20/commits/947f06c3e3e0b7538267c24b904c085ff1d34d00